src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -753,6 +747,23 @@ class ConsentGroupReview extends Component {
     });
   };
 
+  unlinkSampleCollection = (sampleCollection) => () => {
+    ConsentGroup.unlinkSampleCollection(this.props.serverURL, sampleCollection.id).then(
+      () => {
+        this.init()
+      }).catch(error => {
+        this.setState({}, () => { throw error })
+      })
+  };
+
+  handleRedirectToInfoLink = (consentCollectionId, projectKey) => {
+    return [this.props.serverURL, "infoLink", "showInfoLink?cclId=" + consentCollectionId + "&projectKey=" + projectKey + "&consentKey=" + this.props.consentKey].join("/");
+  };
+
+  handleRedirectToProject = (projectKey, projectType) => {
```

Comment:
Similar project-link logic is also in 3 other places:
* `Table.js`
* `ConsentGroupDocuments.js`
* `InfoLink.js`
These should be consolidated into a common utility class.

Reply:
I created a utility class as you suggested. There's a not very pretty method to obtain the controller string from the project type name. This will be removed in Veronica's PR, where we've changed how we redirect to each type of project. Please let me know if this is ok for now.